packages/mgt-chat/src/components/ChatListItem/ChatListItem.tsx

```diff
@@ -174,25 +206,45 @@ export const ChatListItem = ({ chat, myId, onSelected }: IMgtChatListItemProps &
   const removeHtmlPTags = (str: string) => {
     return str.replace(/<\/?p>/g, '');
   };
+  const removeHtmlATags = (str: string) => {
+    return str.replace(/<\/?a[^>]*>/g, '');
+  };
 
   const enrichPreviewMessage = (previewMessage: NullableOption<ChatMessageInfo> | undefined) => {
     let previewString = '';
+    let content = previewMessage?.body?.content as string;
+
+    // handle emojis
+    content = rewriteEmojiContent(content);
+
+    // handle images
+    const graphImageUrlRegex = /<img[^>]+?src=["']https:\/\/graph\.microsoft\.com\/[^"'>]+?["']/;
+
+    const imageMatch = content.match(graphImageUrlRegex);
```

Comment:
This crashes with an error if content is `null | undefined`. You can probably just shortcut out of this method if there is no preview.